src/print.lisp

```diff
@@ -319,14 +319,21 @@
     (simple-format stream res)))
 
 ;;; hash-table printer
+(defun %hash-fn-print-name (form)
+  (let ((name (oget (car form) "name")))
+    (string-downcase (subseq name 0 (position #\- name)))))
+
+(defun %hash-table-count (obj)
+  ;; todo: #j:Object:entries as builtin (method-call |Object| "entries" ,object)
+  (oget (#j:Object:entries (cdr obj)) "length"))
```

Comment:
Can we move `%hash-table-count` to the `hash-table.lisp` module?

It doesn't really belong in the printer.  Also, the printer can use the non-prefixed function `(hash-table-count` ideally,  and hash-table.lisp can define that as an alias for `%hash-table-count` on JSCL only.

That way we can test the printer inside other implementations as well!  that's very useful during development.

Reply:
fixed